.github/workflows/assign-dependabot-reviewer.yml

```diff
@@ -31,7 +35,7 @@ jobs:
 
       - name: Determine and assign reviewer
         env:
-          GH_TOKEN: ${{ secrets.DEPENDABOT_REVIEWER_GH_TOKEN }}
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Instead of using the `deploy-code-org` token, use upgrade the actions workflow permissions using the `permissions` token

Reply:
Turns out this doesn't work - the `GITHUB_TOKEN` does not have all the permissions necessary to assign a team reviewer (needs `repo` access which isn't provided by the token).  See : https://github.com/actions/add-to-project/issues/168